### PR TITLE
feat(addon-charts): `ArcChart` add `activeItemIndex`

### PR DESCRIPTION
**Pull request by waterplea**
## PR Checklist

Please check if your PR fulfills the following requirements:

- [ ] The commit message follows [Conventional Commits](https://www.conventionalcommits.org/en/)
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type

What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Refactoring
- [ ] Code style update
- [ ] Build or CI related changes
- [ ] Documentation content changes

## What is the current behavior?

Closes # 

## Does this PR introduce a breaking change?

- [ ] Yes
- [ ] No

### CODE REVIEW

File: projects/addon-charts/components/arc-chart/arc-chart.component.ts [hunk 1]
```diff
@@ -83,6 +113,10 @@ export class TuiArcChartComponent {
         return WIDTH[this.size];
     }
 
+    isInactive(index: number): boolean {
```
Comment (splincode) [inline projects/addon-charts/components/arc-chart/arc-chart.component.ts:116]: `@tuiPure`?

**Review by splincode — APPROVED** [review]

File: projects/addon-charts/components/arc-chart/arc-chart.template.html [hunk 2]
```diff
@@ -14,9 +14,11 @@
         d="M -70 70 A 100 100 0 1 1 70 70"
     />
     <path
+        #arc
```
Comment (vladimirpotekhin) [inline projects/addon-charts/components/arc-chart/arc-chart.template.html:17]: nit: don't you want to move it on svg to apply hover effect on whole arc ?

**Review by vladimirpotekhin — APPROVED** [review]

File: projects/addon-charts/components/arc-chart/arc-chart.component.ts (hunk 1 above)
Comment (waterplea) [reply projects/addon-charts/components/arc-chart/arc-chart.component.ts:116]: No, it is checking every index.

File: projects/addon-charts/components/arc-chart/arc-chart.template.html (hunk 2 above)
Comment (waterplea) [reply projects/addon-charts/components/arc-chart/arc-chart.template.html:17]: I tried both and it felt better to highlight only when you hover over the filled part.

**Pull request merged**